stable/datadog/templates/deployment.yaml

```diff
@@ -1,25 +1,37 @@
+{{- if .Values.deployment.enabled }}
 {{- if .Values.datadog.apiKey }}
-{{- if .Values.datadog.collectEvents }}
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: {{ template "fullname" . }}-events
+  name: {{ template "fullname" . }}
   labels:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
 spec:
-  replicas: 1
+  replicas: {{ .Values.deployment.replicas }}
   template:
     metadata:
       labels:
-        app: {{ template "fullname" . }}-events
-      name: {{ template "fullname" . }}-events
+        app: {{ template "fullname" . }}
+      name: {{ template "fullname" . }}
     spec:
       containers:
-      - name: {{ .Chart.Name }}-events
+      - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         resources:
 {{ toYaml .Values.resources | indent 12 }}
+        ports:
+        - containerPort: 8125
```

Comment:
> The first reason is money. Each team is a separate organization with it's own Datadog account and API key. They are responsible for their budget, as well as for their custom, application-specific dashboards and alerts.

Thanks for explanation. In what situation does that type of team structure exist? At first blush this seems solving for a very specific team needs, not a general use case (as is the goal of Helm charts in general).